src/functionBased/components/TodoItem.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable react/destructuring-assignment */
+import React, { useState, useEffect } from 'react';
+import { FaTrash } from 'react-icons/fa';
+import styles from './TodoItem.module.css';
+
+const TodoItem = (todos) => {
+  const [editing, setEditing] = useState(false);
+
+  const handleEditing = () => {
+    setEditing(true);
+  };
+
+  const handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      setEditing(false);
+    }
+  };
+
+  useEffect(() => () => {
+    console.log('Cleaning up...');
```

Comment:
- [x] Good job 👌 but there are some issues for you 😊 to do, kindly remove the console from the uploading it will be better😊if you did that.👌

Reply:
Thank you for pointing this out.